### PR TITLE
Implement MediaViewer Plugin

### PR DESCRIPTION
**Pull request by MuhweziDeo**
Implement media viewer plugin for images, video, pdf files

### CODE REVIEW

**Review by allyoucanmap — COMMENTED** [review]
@MuhweziDeo The implementation seems good, could you add some testing to this new components?

**Comment by MuhweziDeo** [comment]
> @MuhweziDeo The implementation seems good, could you add some testing to this new components?
Okay I will

File: geonode_mapstore_client/client/themes/geonode/scss/geonode.scss [hunk 1]
```diff
@@ -0,0 +1,310 @@
+/*
```
Comment (MuhweziDeo) [inline geonode_mapstore_client/client/themes/geonode/scss/geonode.scss:1]: @allyoucanmap maybe I did something wrong when updating my branch but is this file needed

Comment (allyoucanmap) [reply geonode_mapstore_client/client/themes/geonode/scss/geonode.scss:1]: @MuhweziDeo I think you can remove this file

**Review by allyoucanmap — APPROVED** [review]
